src/cts/zcl_abapgit_transport_objects.clas.abap

```diff
@@ -1,24 +1,24 @@
 CLASS zcl_abapgit_transport_objects DEFINITION
   PUBLIC
   FINAL
-  CREATE PUBLIC .
+  CREATE PUBLIC.
 
   PUBLIC SECTION.
 
     METHODS constructor
       IMPORTING
-        !it_transport_objects TYPE zif_abapgit_definitions=>ty_tadir_tt .
+        it_transport_objects TYPE zif_abapgit_definitions=>ty_tadir_tt.
     METHODS to_stage
       IMPORTING
-        !io_stage           TYPE REF TO zcl_abapgit_stage
-        !is_stage_objects   TYPE zif_abapgit_definitions=>ty_stage_files
-        !it_object_statuses TYPE zif_abapgit_definitions=>ty_results_tt
+        io_stage           TYPE REF TO zcl_abapgit_stage
+        is_stage_objects   TYPE zif_abapgit_definitions=>ty_stage_files
+        it_object_statuses TYPE zif_abapgit_definitions=>ty_results_tt
       RAISING
-        zcx_abapgit_exception .
+        zcx_abapgit_exception.
   PROTECTED SECTION.
   PRIVATE SECTION.
 
-    DATA mt_transport_objects TYPE zif_abapgit_definitions=>ty_tadir_tt .
```

Comment:
please dont change this part, it makes it more difficult to review the PR

Reply:
Ok, yeah I have some abap-cleaner settings for these things.

Reply:
dont get me started about abap-cleaner...

Reply:
I would like to hear, maybe in another forum :)